### PR TITLE
Add handler for `/session/:sessionId/element/active` endpoint

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
There are both W3C and non-W3C routes

### CODE REVIEW

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/FindActive.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import io.appium.espressoserver.lib.model.ViewAttributesEnum;
+
+public class FindActive implements RequestHandler<AppiumParams, Element> {
+
+    @Override
+    public Element handle(AppiumParams params) throws AppiumException {
+        final Locator locator = new Locator();
+        locator.initUriMapping(params);
+        locator.setUsing(Strategy.XPATH);
+        locator.setValue(String.format("//*[@%s='true']", ViewAttributesEnum.FOCUSED.toString()));
```
Comment (vmaxim) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/FindActive.java]: Maybe it's better to use  `View.isFocused()` instead this trick?

Comment (mykola-mokhnach) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/FindActive.java]: It would generate much more code than this

Comment (vmaxim) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/FindActive.java]: Few extra lines better than magic locators. Also performance will be increased significantly.

**Comment by mykola-mokhnach** [comment]
@vmaxim how do you find the updated implementation?

**Comment by vmaxim** [comment]
@mykola-mokhnach Nice. Easy to read, easy to understand and pure espresso.

**Pull request merged**